TestRunners/AnnotationTestRunner/run_test.py

```
@@ -0,0 +1,51 @@
+import os
+import sys
+
+if os.path.split(os.getcwd())[1] != "build":
+    print(f"WARNING: not in build dir; cwd is {os.getcwd()}")
+    sys.exit(1)
+
+sys.path.insert(0, '..')
+from TestRunners.FlagsManager import FlagsManager
+from AnnotationTestFilesManager import AnnotationTestFilesManager
+from AnnotationTestRunner import AnnotationTestRunner
+
+
+if __name__ == "__main__":
+    '''
+    Usage Examples: 
+    python3 ../run_at.py [file name] [flags]
+    python3 ../run_at.py -h to get help
+
+    python3 ../run_at.py includes_test
+    python3 ../run_at.py includes_test --no-run-for subdir/runner.c
+    python3 ../run_at.py includes_test --only-run-for call_structs.c
+    python3 ../run_at.py includes_test --no-run-for subdir/runner.c
+    '''
+
+    flags_manager = FlagsManager(
+        "python3 ../run_at.py [required test folder name] [optional flags]")
+    flags_manager.add_flag(
+        "h", "help",
+        "Show help"
+    )
+
+    flags_manager.add_flag(
+        "r", "no-run-for",
+        "Does not check annotations for specified c file. Can be called multiple times. c file must be specified in the succeeding argument. Combined with --only-run-for may result in undesirable output.",
```

Should we throw an exception here if both flags are passed?